pages/settings/wallets/ConnectWallet.tsx

```diff
@@ -0,0 +1,281 @@
+        ],
+        metadata: null,
+      });
+
+      console.info("createConnection response", response);
+
+      const newUrl = `nostr+walletconnect://${response.wallet_pubkey}?secret=${bytesToHex(
+        secretKey,
+      )}&relay=${nwcClient.relayUrl}`;
+
+      setNwcUrl(newUrl);
+      setRedirectCountdown(5);
+    } catch (error) {
+      console.error(error);
+      errorToast(error);
+    }
+    setLoading(false);
+  };
+
+  return (
+    <>
+      <Screen
+        title={nwcUrl ? "Wallet Connected" : "Connect Wallet"}
+        right={() => (
+          <TouchableOpacity
+            onPressIn={() => {
+              if (router.canDismiss()) {
+                router.dismissAll();
+              }
+              router.replace("/");
+            }}
+            className="-mr-4 px-6"
+          >
+            <XIcon className="text-muted-foreground" width={24} height={24} />
+          </TouchableOpacity>
+        )}
+      />
+      <View className="flex-1 justify-center items-center gap-8 p-6">
+        <View className="flex flex-row items-center justify-center">
+          <View className="flex items-center">
+            <View className="shadow">
+              <Image
+                source={require("../../../assets/hub.png")}
+                className="my-4 rounded-2xl w-20 h-20"
+              />
+            </View>
+            <Text className="text-xl font-semibold2">Alby Hub</Text>
+          </View>
+
+          <View className="z-[-1] relative w-36 h-16 mb-4 items-center justify-center">
+            <Animated.Image
+              resizeMode="contain"
+              source={require("../../../assets/left-plug.png")}
+              className="absolute w-20 h-20"
+              style={{
+                left: -10,
+                transform: [
+                  {
+                    translateX: leftPlugAnim,
+                  },
+                ],
+              }}
+            />
+            <Animated.Image
+              resizeMode="contain"
+              source={require("../../../assets/right-plug.png")}
+              className="absolute w-20 h-20"
+              style={{
+                right: -20,
+                transform: [
+                  {
+                    translateX: rightPlugAnim,
+                  },
+                ],
+              }}
+            />
+          </View>
+
+          <View className="flex items-center">
+            <View className="shadow">
+              <Image
+                source={{ uri: appicon }}
+                className="my-4 rounded-2xl shadow-md w-20 h-20"
+              />
+            </View>
+            <Text className="text-xl font-semibold2">{appname}</Text>
+          </View>
+        </View>
+        {nwcUrl ? (
+          <View className="flex-1 w-full justify-center items-center">
+            <Tick />
+            <Text className="my-4 text-lg text-center text-foreground">
+              Connected! You're being redirected in {redirectCountdown}{" "}
```

Comment:
this should only show if there is a callbackUrl set. Otherwise there should just be an OK button to go back to the home page

Reply:
Yup returning to Home sounds good :+1:

Reply:
done